students/wonjae/users/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+# Create your models here.
+
+class User(models.Model):
+    name = models.CharField(max_length=20)
+    email = models.CharField(max_length=200)
+    pw = models.CharField(max_length=100)
```

Comment:
변수명은 줄여서 선언하시지 마시고, 정확하게 만들어주세요.
```suggestion
    password = models.CharField(max_length=100)
```